libs/cart/state/src/guards/non-empty-cart/non-empty-cart.guard.ts

```diff
@@ -0,0 +1,38 @@
+import { CanActivate, Router } from '@angular/router';
```

Comment:
The file name and path needs to be changed to `cart-item` from `non-empty-cart`

Reply:
good catch, changed